ow_lander/src/ow_actions/arm.py

```diff
@@ -0,0 +1,106 @@
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
```

Comment:
A short explanation of what this file is for would be helpful, especially since the filename is so simple.  On that note, should this file be named ArmActionMixin.py?  (I don't know Python naming conventions).

Reply:
I could stand to add more file descriptions.

Aside from requiring snake_case and brevity where possible there aren't much in the way of module naming conventions. `arm_action_mixin.py` works fine for me. 

Alternatively, I could put all of the arm interface classes (ArmTrajectoryPlanner, ArmTrajectoryExecutor, and ArmActionMixin) under a single module named `arm.py`. Thoughts?